agent/unit-test/containeroptionparser_unittest.cpp

```diff
@@ -63,6 +63,40 @@ TEST_F(ContainerOptionParserTest, parseConfigNice2) {
     ASSERT_FALSE(m_options->enableWriteBuffer());
 }
 
+TEST_F(ContainerOptionParserTest, parseConfigNice3) {
```

Comment:
On a glance, this looks like a good candidate for parametrization. If not, then I definitely think they should have proper names and comments on each of them what's being tested.